### ISSUE TITLE
Failure processing application bundle

### ISSUE THREAD

**Issue by yochananmarqos** [withinboredom/system-76-keyboards#27]
Whether building 1.1.0  from source, extracting the files from `manual.tgz` or `package.deb`, running `/usr/bin/keyboard-colors` results in the following on Manjaro GNOME:

```
Failure processing application bundle; possible file corruption.
I/O failure reading contents of the bundle.
A fatal error was encountered. Could not extract contents of the bundle
```

By the way, what .NET SDK version do you use to build?

**Comment by withinboredom** [comment]
> By the way, what .NET SDK version do you use to build?

I believe it was .net core 3.1ish. I've been meaning to get back around to this and update it / clean it up some. I just haven't had much in the way of free time and it still runs on my machine with pop os 20.04 so it hasn't been much of a priority.

**Comment by withinboredom** [comment]
I wonder if it's related to this issue fixed in 3.1: https://github.com/dotnet/runtime/issues/3787

**Comment by yochananmarqos** [comment]
I just realized what the problem was. By default `makepkg` strips symbols from binaries/libraries. Disabling it solved the problem.
